test_util/helpers.py

```diff
@@ -37,6 +37,81 @@ def authenticate(self, cluster):
         logging.info('Authentication successful')
 
 
+class ApiClient:
+
+    def __init__(self, cluster, user, api_base, default_headers=None, ca_cert_path=None):
+        self.cluster = cluster
+        self.user = user
+        self.api_base = api_base
+        if default_headers is None:
+            default_headers = dict()
+        self.headers = default_headers
```

Comment:
nit: I'd rather be consistent on this naming everywhere, either always default_headers or always headers.